projects/plugins/jetpack/extensions/blocks/sharing-button/variations.js

```diff
@@ -89,6 +90,12 @@ export const variations = [
 		title: 'Nextdoor',
 		icon: <SocialIcon icon={ 'nextdoor' } size={ 24 } />,
 	},
+	{
+		name: 'sms',
+		attributes: { service: 'sms', label: 'SMS' },
```

Comment:
I think we may have to make "SMS" translatable, it may not be called that way in every language.